app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+class InvalidCommandFormat(Exception):
+    pass
+
+
+def copy_file(command: str) -> None:
+    command = command.split()
```

Comment:
it's better to unpack all values into descriptive variables

command, old_file, copy_file = command.split()

now its not clear what command[1] and command[2] refers to.